src/main/scala/jtag/Utils.scala

```diff
@@ -43,8 +45,7 @@ object NegativeEdgeLatch {
 /** A module that counts transitions on the input clock line, used as a basic sanity check and
   * debug indicator clock-crossing designs.
   */
-class ClockedCounter(modClock: Clock, counts: BigInt, init: Option[BigInt])
-    extends Module(override_clock=Some(modClock)) {
+class ClockedCounter(counts: BigInt, init: Option[BigInt]) extends Module {
```

Comment:
Maybe could be cleaned up further - but doesn't need to be this PR, could be a future one.